src/components/RequestGroupComposition/RequestGroup.vue

```diff
@@ -265,6 +266,10 @@ export default {
         return 0;
       }
     },
+    mosaicMaxNumPointings: {
+      type: Number,
+      default: 100
```

Comment:
seems bad to have the default in multiple places... should we just have it in the RequestGroup and pass it down to the Request?

Reply:
Yeah good point... I placed it everywhere in case people wanted to pull in and use components that aren't RequestGroup and they'd have the same default, but that's not a common use case and yeah you're right this will probably end up being confusing at some point. I'll change this.